docs/manual/manual_redundant_throttle.rst

```diff
@@ -0,0 +1,113 @@
+    or hobby as redundant throttle sources.
+
+Configuration Example 1
+##########################
+
+Suppose your module supports DroneCAN, IQUART, and hobby inputs. 
+
+You configure the following:
+
+- IQUART's priority to 3 and DroneCAN's to 1 (the module will automatically set hobby's to 2 ensuring that there are no two matching priorities)
+- Flight controller outputting both :ref:`IQUART Flight Controller Interface commands <controlling_ifci>` as well as DroneCAN commands
+- Throttle source timeout configured to 0.5 seconds
+
+The sequence diagram below illustrates how your module reacts to received throttle commands as well as how it deals with switching between sources.
+
+.. image:: ../_static/manual_images/redundant_throttle/iquart_prio_example.png
+    :width: 50%
+
+Note that the first DroneCAN throttle is applied to the motor since no IQUART messages had been received before. Then, when both an IQUART and DroneCAN message are received, 
+the IQUART throttle is applied. Last, DroneCAN throttles are only applied again once the throttle timeout is reached after IQUART disconnects. If for any reason, the module begins receiving 
+IQUART throttles again at this point, the module will immediately start accepting those as IQUART has a higher priority.
+
+Configuration Example 2
+##########################
+
+Suppose your module supports DroneCAN, IQUART, and hobby inputs. 
+
+You configure the following:
+
+- DroneCAN's priority to 3, IQUART to 0, hobby to 0
+- Flight controller outputting both :ref:`hobby_dshot` as well as DroneCAN commands
+- Throttle source timeout configured to 0.1s
+
+.. image:: ../_static/manual_images/redundant_throttle/dshot_ignored_ex.png
+    :width: 70%
+
+In this example, the module is configured to ignore all hobby and IQUART throttle messages. So, all received DSHOT throttles are dropped, and throttles are only applied once DroneCAN 
+throttles are received. An important note from this example is that the module will reach its :ref:`propeller motor control timeout <manual_timeout>` even if the 
+flight controller continues to send DSHOT throttles. As they are not processed by the throttle source manager, DSHOT throttles cannot be used to reset the timeout timer.
+
+********************************
+Redundant Throttle and Arming
+********************************
+
+All throttle commands received by the throttle source manager are sent to your module's :ref:`arming handler <manual_advanced_arming>`, and are subject to all constraints defined by the arming handler. 
+For example, suppose you have configured IQUART to be the highest priority source, and are sending IQUART throttle commands via the :ref:`Vertiq Testing Tool <vertiq_testing__guide>`. At the same 
+time, you are transmitting DroneCAN throttles with the :ref:`DroneCAN GUI tool <dronecan_gui_basics>`. In this example, your arming handler is set to arm on throttle in the range 0-7.5% on 10 

```

Comment:
the "set to arm on throttle in the range...." may be better as "set to arm on throttles". The parameter is called arm on throttle, but in this sentence we are arming on throttles in the range 0 to 7.5% I think